pkgs/development/tools/zigmod/default.nix

```diff
@@ -0,0 +1,159 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchgit
+, zig
+}:
+
+let
+  deps = [
+    {
+      url = "https://gist.github.com/nektro/d468fea84f8217e4c26ee8fbeeea38cc";
+      rev = "0911d65210fe14467fa73afd5b09d781a3995adc";
+      sha256 = "sha256-G62R6rYd2NwMnayyuzZeMLOTb+LYOKeN7ithy8YYTwI=";
+    }
+    {
+      url = "https://github.com/MasterQ32/zig-uri";
```

Comment:
> NOTICE: THIS LIBRARY IS DEPRECATED!
>    This library is now part of zig std library, available as std.Uri since [87b2234](https://github.com/ziglang/zig/commit/87b223428a8953b6af9bea61ff4cc905821c0557).

The upstream is not even worried about using the Zig stdlib, but source-vendors an outdated, archived repo? Seriously?